instructors.md

```diff
@@ -337,6 +228,132 @@ a substantial fraction won't be able to navigate to the right directory without
+    *   We tried a GitHub-backed blog in the Winter 2015 class.
+        It didn't work nearly as well:
+        a third of the participants found it extremely frustrating,
+        and post-publication commentary was awkward.
+    *   We tried Piazza in the Fall 2015 class.
+        It was better than GitHub,
+        but still not as good as a simple WordPress blog.
+        In particular,
+        it was hard to find things once there were more than a dozen homework categories.
+
+### Two-Day In-Person
+
+This was the second method we tried.
+The biggest change was the introduction of recorded teaching exercises.
+
+*   Several times during the training,
+    participants are divided into groups of three
+    and asked to teach a short lesson (typically 2-3 minutes long).
+    In turn,
+    one person is the teacher, one the audience, and one the videographer,
+    who records the teacher using a handheld device such as a phone.
+    Group members then rotate roles:
+    the teacher becomes the listener, the listener records, and the videographer teaches.
+    Once all three have finished teaching,
+    the group reviews all three videos,
+    and everyone gives feedback on everyone (including themselves).
+    This feedback then goes into the Etherpad for discussion.
+
+*   It's important to record all three videos and then watch all three:
+    if the cycle is teach-review-teach-review,
+    the last person to teach runs out of time.
+    Doing all the reviewing after all the teaching
+    also helps put a bit of distance between the teaching and the reviewing,
+    which makes the exercise slightly less excruciating.
+
+*   This exercise only works if there are breakout rooms available:
+    if everyone is trying to record in the same room,
+    the audio cross-talk makes the recordings unintelligible.
+
+*   We use Etherpad for in-person training,
+    both for note-taking and for posting exercise solutions and feedback on recorded lessons
+    Questions and discussion are done aloud.
+
+### Two-Day Online With Groups
+
+*   We use Google Hangouts and Etherpad as in the multi-week version.
+    Each group of learners is together in a room using one camera and microphone,
+    rather than each being on the call separately.
+    We have found that having good audio matters more than having good video,
+    and that the better the audio,
+    the more learners can communicate with the instructor and other rooms by voice
+    rather than by using the Etherpad chat.
+
+*   We do the video lecture exercise as in the two-day in-person training.  
+
+<a name="schedule"></a>
+## Schedule
+
+The schedule for the course is evolving along with the content,
+but here's what we do in a typical two-day event:
+
+| Day 1 | 09:00 | Welcome: key concepts and training goals            |
+|       | 09:30 | Novice mind-set; formative vs. summative assessment |
+|       |       | *Exercise: multiple choice questions*               |
```

Comment:
This is also about mental models. The important bit is that we're trying to help build people correct mental models (achievable in 2 days) rather than make them expert programmers or hammer 597464 bash commands into their memory.